### PR TITLE
Feature/improvise test posts

### PR DESCRIPTION
**Pull request by Joralmo**
# Description
Include a summary of the changes you've made here.

## Type of changes
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that causes existing functionality not to work as expected)
 - [ ] Documentation update (if none of the other choices apply)

## Checklist
 - [x] I have read the [CONTRIBUTING](https://github.com/NoEMEC/fakedata/.github/CONTRIBUTING.md) document
 - [x] I have added necessary documentation (if appropriate)

## Further comments
Any other notes (if any).

### CODE REVIEW

File: src/v1/controllers/posts/post.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            const posts = await service.getPosts();
+            expect(posts instanceof Array).toBe(true);
+            done();
+        });
+    });
+
+    describe('createPosts', () => {
+        it('should return a new post', async done => {
+            const post = await service.createPost(testPost);
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (EndersonPro) [inline src/v1/controllers/posts/post.service.spec.ts]: Si lo haces de la forma clásica?, con el el delete post._id para seguir con el estándar de eslint y no tener que omitir lineas.

Comment (Joralmo) [reply src/v1/controllers/posts/post.service.spec.ts]: @EndersonPro ready

**Review by EndersonPro — APPROVED** [review]

**Review by EndersonPro — APPROVED** [review]

File: src/v1/controllers/posts/post.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+const mongoMemory = new MongoMemoryServer();
+let connection: Connection;
+
+const testPost = {
+    title: 'Test title',
+    body: 'Test body',
+    userId: 'randomId',
+};
+
+// const testPosts = [
```
Comment (krrskl) [inline src/v1/controllers/posts/post.service.spec.ts]: Esto debería continuar aquí?

**Review by krrskl — APPROVED** [review]

File: src/v1/controllers/posts/post.service.spec.ts (hunk 2 above)
Comment (Joralmo) [reply src/v1/controllers/posts/post.service.spec.ts]: fixed

**Pull request merged**